lib/logic/entities/Entity.js

```diff
@@ -0,0 +1,45 @@
+const yaml   = require('js-yaml');
+
+class Entity {
+    toDefault() {
```

Comment:
please add here ability to retrieve the raw returned body (i think it is the info field)
please try to explain it better for the developers